src/routes/docs/products/functions/templates/+page.markdoc

```diff
@@ -0,0 +1,128 @@
+---
+layout: article
+title: Templates
+description: Learn about Appwrite Functions' templates that let you jump start function development to extend your Appwrite APIs.
```

Comment:
Could the `'` in this description cause problems with the parser? I vaguely remember an issue like this, please check it doesn't result in a server error

Reply:
I'll check :)